src/share/poudriere/image_zfs.sh

```diff
@@ -38,10 +38,11 @@ _zfs_writereplicationstream()
 
 zfs_check()
 {
+	zroot="${ZFS_POOL_NAME}.$(jot -r 1 1000000000)"
```

Comment:
While here, what happens if someone overrides (e.g., via a pre-script) this `zfs_check()` function and "forgets" to set `zroot`?